core/linux-tools/build

```diff
@@ -0,0 +1,118 @@
+#!/bin/sh -uxe
+
+export KBUILD_BUILD_HOST=`hostname`
+export KBUILD_BUILD_USER=`id -nu`
+export CC=gcc
+export CXX=g++
+export BUILD_LD="ld.bfd"
+
+pkgver="$2"
+
+echo ':: cpupower'
+pushd tools/power/cpupower
+make CC="${CC}" CXX="${CXX}" VERSION=$pkgver-1
+make CC="${CC}" CXX="${CXX}" \
+  DESTDIR="$1" \
+  sbindir='/usr/bin' \
+  libdir='/usr/lib' \
+  mandir='/usr/share/man' \
+  docdir='/usr/share/doc/cpupower' \
+  install install-man
+popd
+
+echo ':: x86_energy_perf_policy'
+pushd tools/power/x86/x86_energy_perf_policy
+make CC="${CC}" CXX="${CXX}"
+install -Dm 755 x86_energy_perf_policy "$1/usr/bin/x86_energy_perf_policy"
+install -Dm 644 x86_energy_perf_policy.8 "$1/usr/share/man/man8/x86_energy_perf_policy.8"
+
+popd
+
+#echo ':: usbip'
```

Comment:
Any reason why `usbip`, `tmon` `cgroup_event_listener` and `turbostat` are commented?